modules/grabber.js

```diff
@@ -0,0 +1,3 @@
+export default function grabber(e) {
+  return document.getElementById(e);
```

Comment:
- [ ] Same here. I would recommend using the arrow syntax for the function. As it is requirement to use ES6 syntax as much as you can to get familiar with.